src/ethereum/ethereumTypes.ts

```diff
@@ -18,6 +18,7 @@ import { EdgeSpendInfo } from 'edge-core-js/types'
 import {
   asSafeCommonWalletInfo,
   Dapp,
+  WalletConnectPayload,
```

Comment:
Import is unused. How did this commit pass the linter?

Reply:
That shouldn't have even been a commit. I combined the first two